src/witnet/request.js

```diff
@@ -48,6 +49,12 @@ class Request {
 
     return this
   }
+  setCollateral (collateral) {
+    if (collateral != 0){
```

Comment:
Why guarding against `0`?

Reply:
Because the node was not able to serialize a 0. But I will guard against anything that is less than the minimum collateral